pkg/queue/stats_reporter.go

```diff
@@ -37,13 +38,19 @@ const (
 	// will be dropped if LastValue aggregation is used for a view.
 	ReporterReportingPeriod = time.Second
 
-	operationsPerSecondN       = "operations_per_second"
-	averageConcurrentRequestsN = "average_concurrent_requests"
+	operationsPerSecondN        = "operations_per_second"
+	proxiedOperationsPerSecondN = "proxied_operations_per_second"
+	averageConcurrentRequestsN  = "average_concurrent_requests"
+	averageProxiedConcurrencyN  = "average_proxied_concurrency"
 
 	// OperationsPerSecondM number of operations per second.
 	OperationsPerSecondM Measurement = iota
 	// AverageConcurrentRequestsM average number of requests currently being handled by this pod.
 	AverageConcurrentRequestsM
+	// ProxiedOperationsPerSecondM number of proxied operations per second.
+	ProxiedOperationsPerSecondM
+	// AverageProxiedConcurrencyM average number of proxied requests currently being handled by this pod.
+	AverageProxiedConcurrencyM
```

Comment:
Same comment, should we have the same naming scheme?

Reply:
Done.